pygromos/files/gromos_system/gromos_system.py

```diff
@@ -220,7 +220,7 @@ def __init__(self, work_folder: str, system_name: str, in_smiles: str = None,
         if in_cnf_path is None and type(self.mol) == Chem.rdchem.Mol and self.mol.GetNumAtoms() >= 1:
             self.cnf = Cnf(in_value=self.mol)
             #TODO: fix ugly workaround for cnf from rdkit with GROMOS FFs
-            if self.Forcefield.name == "2016H66" or self.Forcefield.name == "54A7":
+            if self.Forcefield.name == "2016H66" or self.Forcefield.name == "54A7" and os.path.exists(self._gromosPP_bin_dir):
```

Comment:
this a nice fix for the current issue here.
maybe we should add functionality like: 
- check_presence_gromos_bin_dir (path_to_bin_dir)-> bool
- check_presence_gromos_bin (binary_name) -> bool   #this function add the bin_dir or PATH var content to see if binary is accessible.

results could be stored in gromosPP_accisble:bool

What do you think @MTLehner @pultar